### DISCUSSION TITLE
When I am controlling multiple devices, I want each controlled device to be a separate active window. How do I set it up?

### DISCUSSION THREAD

**Discussion by dabai998** [rustdesk/rustdesk#5061] [Feature Request]
Now there are multiple devices in one window, and when viewing device content, each window needs to be switched. It is hoped that each controlled end can be divided into separate windows, so that there is no need to switch windows when viewing device content

**Comment by rustdesk** [comment]
Next version.

**Reply by dabai998** [reply]
Looking forward to the new version very much, you can see the content of multiple windows without switching, it will be very convenient, thank you
